stylesheets/_utility.states.scss

```diff
@@ -32,6 +32,15 @@ $danger-to-text-color: black;
     }
 }
 
+.has-error:not(.form__group),
+.has-danger:not(.form__group) {
+    background-color: lighten(color(danger), 20) !important;
+
+    &:hover {
+        background-color: lighten(color(danger), 17) !important;
+    }
+}
+
```

Comment:
Welcome back to the party pal!
![200](https://user-images.githubusercontent.com/9677023/39056891-c7010f2c-44af-11e8-9fcc-988fa996f179.gif)